types/api/cpa.d.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Same here, it would be beneficial to rename it to customProfileAttributes.ts for readability.  Easier to search, etc.